source/amf.cpp

```diff
@@ -153,7 +153,7 @@ Plugin::AMD::AMF::AMF()
 
 	/// Blacklist Drivers with older SDK.
 	if (m_AMFVersion_Runtime < AMF_FULL_VERSION) {
-		PLOG_ERROR("The detected AMF runtime is too old, please update your drivers.");
+		PLOG(LOG_ERROR, "The detected AMF runtime is too old, please update your drivers.");
```

Comment:
I have no idea why this macro expansion doesn't work but I also don't care. It's the only instance where this macro is used.

Reply:
If this is the only place where `PLOG_ERROR` is used, do we need to keep the macro definition? Or are you just being safe by keeping it?

Reply:
Nevermind I changed it and when doing a clean build it does compile now.